tensorflow/core/kernels/sparse_xent_op_test.cc

```diff
@@ -41,40 +41,52 @@ static Graph* SparseXent(int batch_size, int num_classes) {
   return g;
 }
 
-#define BM_SparseXentDev(BATCH, CLASS, DEVICE)                          \
-  static void BM_SparseXent##_##BATCH##_##CLASS##_##DEVICE(int iters) { \
-    testing::ItemsProcessed(static_cast<int64>(iters) * BATCH * CLASS); \
-    test::Benchmark(#DEVICE, SparseXent(BATCH, CLASS)).Run(iters);      \
-  }                                                                     \
-  BENCHMARK(BM_SparseXent##_##BATCH##_##CLASS##_##DEVICE);
+#define BM_SparseXentDev(BATCH, CLASS, DEVICE, DTType)                     \
+  static void BM_SparseXent##_##BATCH##_##CLASS##_##DEVICE_##DTType(       \
```

Comment:
Nit: Dtype is more common. Also, let's use all upper case.
```suggestion
  static void BM_SparseXent##_##BATCH##_##CLASS##_##DEVICE_##DTYPE(       \
```